src/app/proposals/create/CreateBuilderProposalForm.tsx

```diff
@@ -47,11 +48,7 @@ export const CreateBuilderProposalForm: FC = () => {
   const router = useRouter()
   useVotingPowerRedirect()
   const { setMessage } = useAlertContext()
-  const { onCreateBuilderWhitelistProposal, isPublishing, error } = useCreateBuilderWhitelistProposal()
-  if (error) {
-    setMessage(TX_MESSAGES.proposal.error)
-    console.error('🐛 Error writing to contract:', error)
-  }
+  const { onCreateBuilderWhitelistProposal, isPublishing } = useCreateBuilderWhitelistProposal()
```

Comment:
@DinizSa Why don't we check for errors when creating the Builder whitelist proposal?